src/applications/vaos/services/mocks/featureFlags.js

```diff
@@ -1 +0,0 @@
-../../utils/featureFlags.js
```

Comment:
This [file](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/services/mocks/featureFlags.js) is confusing me a little. It was added in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/26695). @vbahinwillit do you know why this was added? I want to make sure I'm not deleting something that might be used other than being imported by other files.